internal/provider/resource_integration.go

```diff
@@ -222,6 +222,10 @@ func (r *integrationResource) Create(ctx context.Context, req resource.CreateReq
 
 	appsType := req.Plan.Schema.GetBlocks()["application"].(schema.SetNestedBlock).NestedObject.Type()
 	parsedApps, errDiag := types.SetValueFrom(ctx, appsType, parsedApplications)
+	if errDiag.HasError() {
+		resp.Diagnostics.Append(errDiag...)
+		return
+	}
```

Comment:
Why did you add this? The errDiag is being checked, only after being added to the response.Diagnostics.

Reply:
Unintentionally, I think I was fixing issues with `golangci-lint`.
Let me revert the changes.